backend/REST-API.js

```diff
@@ -1,5 +1,11 @@
 const express = require('express')
 const router = express.Router()
+const axios = require('axios');
+
+router.get('/api/cta', async (req, res) => {
+    const response = await axios.get('https://ctabustracker.com/bustime/api/v3/getvehicles?key=gbNab8LFcBWDx5qmtc8Q8dfjb&rt=172&format=json')
+    res.json(response.data)
+})
```

Comment:
seems this might have been left over from a previous experiment, can u discard this change?